tests/testdata/codegen/aws-eks-pp/python/aws-eks.py

```diff
@@ -18,10 +18,10 @@
     })
 eks_route_table = aws.ec2.RouteTable("eksRouteTable",
     vpc_id=eks_vpc.id,
-    routes=[{
-        "cidr_block": "0.0.0.0/0",
-        "gateway_id": eks_igw.id,
-    }],
+    routes=[aws.ec2.RouteTableRouteArgs(
```

Comment:
This doesn't look right to me. First, AWS has TypedDicts defined. But more importantly, all existing SDKs support accepting dicts (untyped), so programgen can always pass args for nested objects as dicts. And we want to pass dicts going forward.

Reply:
So should we just always pass dicts in program gen and never use the named classes?

Reply:
I think we should always pass dicts, unless the schema explicitly has only `classes` specified.